compliance_checker/ioos.py

```diff
@@ -1081,6 +1081,49 @@ def check_instrument_variables(self, ds):
 
         return results
 
+    def check_qartod_variables_flags(self, ds):
+        """
+        https://github.com/ioos/ioos-metadata/pull/16/commits/c1833adce183d67e0f05edaa0b52c34de431214f
```

Comment:
Is there a section of a document that could be referenced instead?

Reply:
It's not currently merged yet: https://github.com/ioos/ioos-metadata/pull/16

Reply:
I just took care of it. You can link to this anchor tag: https://ioos.github.io/ioos-metadata/ioos-metadata-profile-v1-2.html#quality-controlqartod